schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+        ]
+
+      },
+      "inactive_licenses": {
+        "title": "Inactive licenses",
+        "description": "Historical licenses that are no longer selectable upon submission to arXiv.",
+        "type": "string",
+        "enum": [
+          "http://arxiv.org/licenses/assumed-1991-2003/",
+          "http://creativecommons.org/licenses/by/3.0/",
+          "http://creativecommons.org/licenses/by-nc-sa/3.0/",
+          "http://creativecommons.org/licenses/publicdomain"
+        ]
+      },
+      "person": {
+        "type": "object",
+        "properties": {
+          "full_name": {
+            "description": "The fullest representation of the person's name available in arXiv metadata.",
+            "type": "string",
+            "minLength": 1
+          },
+          "last_name": {
+            "description": "The family or surname part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "first_name": {
+            "description": "The personal or forename part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "suffix": {
+            "description": "The suffix part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Jr.",
+              "Sr.",
+              "I",
+              "II",
+              "III"
+            ]
+          },
+          "affiliations": {
+            "title": "List of affiliations",
+            "description": "Institutional affiliations",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/affiliation"
+            },
+            "minItems": 1
+          },
+          "orcid": {
+            "title": "ORCID iD",
+            "allOf": [
+              {
+                "type": "string",
+                "pattern": "^\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]$"
+              },
+              {
+                "type": "string",
+                "$comment": "TODO: consider adding 'orcid' format here; however, some validators hang on unexpected formats. Would need to implement validator using checksum method described in https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier"
+              }
+            ]
+          },
+          "author_id": {
+            "description": "arXiv author identifier, if available.",
+            "oneOf": [
+              {
+                "type": "string",
+                "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$"
+              },
+              {"type": "null"}
+            ]
+          },
+          "is_submitter": {
+            "description": "Indicates whether this person is the submitter of this submission",
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "full_name",
+          "first_name",
```

Comment:
Some people don't have first_names. Some people don't have last_names.

Reply:
I think we can remove the `first_name` requirement but not sure about `last_name`.